tutorials/configure-smtp-relay-tem/index.mdx

```diff
@@ -0,0 +1,146 @@
+---
+meta:
+  title: Configuring an SMTP relay with Transactional Email and Postfix
+  description: This page shows you how to configure an SMTP relay with Transactional Email and Postfix
+content:
+  h1: Configuring an SMTP relay with Transactional Email and Postfix
+  paragraph: This page shows you how to configure an SMTP relay with Transactional Email and Postfix.
+tags: smtp relay transactional-emails postfix send-emails
+categories:
+  - transactional-email
+dates:
+  validation: 2023-07-18
+  posted: 2023-07-18
+---
+
+
+Installing an SMTP relay that sends messages from your servers allows you to be notified if/when there are problems, as the server tends to report them by email.
+
+This also means that if you have installed software on the server, the software can rely on the relay's SMTP rather than on configurations that are not always easy to handle.
```

Comment:
```suggestion
Additionally, this enables software installed on the server to use the SMTP relay instead of dealing with potentially complicated configurations for sending emails directly.
```